velox/exec/MergeJoin.cpp

```diff
@@ -593,6 +599,21 @@ vector_size_t firstNonNull(
 
   return rowVector->size();
 }
+
+bool isNull(
```

Comment:
can you have a more descriptive name and some comments on what exactly this method does?

Reply:
@pedroerp This method doesn't need. Removed.